lib/server.js

```diff
@@ -794,6 +796,10 @@ WalletService.prototype.createTx = function(opts, cb) {
   if (!Utils.checkRequired(opts, ['toAddress', 'amount', 'proposalSignature']))
     return cb(new ClientError('Required argument missing'));
 
+  var feePerKb = opts.feePerKb || 10000;
+  if (feePerKb > 10000)
```

Comment:
Shouldn't this be something like

if (1000 > feePerKb > 100000)

Since we want an error if fee per kb is less that MINRELAYTXFEE (1000) and adding a fee up to 1 mBTC should be allowed in case the blocks start filling up.